routes/web.php

```diff
@@ -49,6 +49,11 @@
     });
 });
 
+Route::group(['prefix' => 'beatmappacks'], function () {
```

Comment:
`/beatmapsets/packs`? Also `type` below can be query string instead of part of url? Which then can use `Route::resource()`.